README.md

```diff
@@ -8,6 +13,29 @@
 
 See [`why_the_name_gofannon.md`](./why_the_name_gofannon.md) for the rich story on why we chose to honor this Celtic Diety
```

Comment:
Add full url to this as well (it's currently relative, change it to https://github.com/The-AI-Alliance/gofannon/blob/main/why_the_name_gofannon.md